streams/src/test/java/org/apache/kafka/streams/InternalTopologyAccessor.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.streams;
+
+import org.apache.kafka.streams.processor.TopologyBuilder;
+import org.apache.kafka.streams.processor.internals.InternalTopologyBuilder;
+
+
+/**
+ * This class is meant for testing purposes only and allows the testing of
+ * topologies by using the  {@link org.apache.kafka.test.ProcessorTopologyTestDriver}
+ */
+public class InternalTopologyAccessor {
+
+    public static InternalTopologyBuilder getInternalTopologyBuilder(Topology topology) {
+        return topology.internalTopologyBuilder;
+    }
+
+    @Deprecated
+    public static InternalTopologyBuilder getInternalTopologyBuilder(TopologyBuilder topologyBuilder) {
```

Comment:
Do we want to add this deprecated function (ditto below for line 172)? I think moving forward we could avoid adding more tests on the deprecated APIs.

Reply:
My thinking was to allow users to write tests, without forcing them to move to the new API, but they will need to migrate eventually, so I'll take it out.